app/upgrades.go

```diff
@@ -114,6 +115,15 @@ var upgrades = map[string]appUpgrade{
 			removeInactiveValidatorDelegations(ctx, app)
 			setupICQ(ctx, app)
 			updateMaxSupply(ctx, app)
+
+			denomToNav := map[string]markertypes.NetAssetValue{
+				"usd.deposit": markertypes.NewNetAssetValue(sdk.NewInt64Coin(markertypes.UsdDenom, 1000), 1),
+				"tp13gc5xe8375msm7y6jhf752e5hcrjypmcpf9wldsjprhnpzlkhqaqawxujf.investment": markertypes.NewNetAssetValue(sdk.NewInt64Coin(markertypes.UsdDenom, 10), 1),
+				"tp1cxuqqyjjf5x66jvlmtvj3juppn370ev7rr5cja3ml65nzhxgvpkszfuvtw.investment": markertypes.NewNetAssetValue(sdk.NewInt64Coin(markertypes.UsdDenom, 10), 33600000),
+				"pm.sale.pool.7v2gsuvnudyfvuig50r3k3":                                      markertypes.NewNetAssetValue(sdk.NewInt64Coin(markertypes.UsdDenom, 19026640), 1),
+			}
+			addMarkerNavs(ctx, app, denomToNav, markertypes.NewNetAssetValue(sdk.NewInt64Coin(markertypes.UsdDenom, int64(150)), 1))
+
```

Comment:
`saffron-rc1` has already been run on testnet. This will need to be in a `saffron-rc2`.

Reply:
Yeah, this isn't valid anymore.  I had this PR in draft mode until @iramiller updated me on what needed to happen.

Reply:
We can't do this in any of the RCx handlers because the data is network specific ...